Flow.Launcher/Languages/en.xaml

```diff
@@ -158,6 +158,7 @@
     <system:String x:Key="SoundEffectTip">Play a small sound when the search window opens</system:String>
     <system:String x:Key="SoundEffectVolume">Sound Effect Volume</system:String>
     <system:String x:Key="SoundEffectVolumeTip">Adjust the volume of the sound effect</system:String>
+    <system:String x:Key="SoundEffectWarning">Flow Launcher can't play sound because "Windows Media Player" doesn't exist on your system. If you need sound, install WMP.</system:String>
```

Comment:
```suggestion
    <system:String x:Key="SoundEffectWarning">Sound effect is unavailable because Windows Media Player is missing. Please install it if you need sound effect.</system:String>
```